packages/typespec-ts/src/utils/modelUtils.ts

```diff
@@ -1446,7 +1446,11 @@ function getPriorityName(schema: Schema, usage?: SchemaContext[]): string {
 }
 
 function getEnumStringDescription(type: any) {
-  if (type.name === "string" && type.enum && type.enum.length > 0) {
+  if (
+    (type.name === "string" || type.alias === "string") &&
```

Comment:
We also have cases for extensible enum for number, should we consider them also?

Reply:
added ut here https://github.com/Azure/autorest.typescript/pull/2661/files#diff-3b7495c06bf1fc41a5e741d3e68d82b771f1052cd2f36773395807e9cb7e4bc4R3638 for boolean type, waiting for typespec team's confirm. Probably won't fix https://github.com/Azure/typespec-azure/issues/1162